deepcell/applications/multiplex_segmentation.py

```diff
@@ -169,6 +183,12 @@ def predict(self,
             np.array: Labeled image
             np.array: Model output
         """
+
+        # create dict to hold all of the post-processing kwargs
+        postprocess_kwargs = {'whole_cell_kwargs': postprocess_kwargs_whole_cell,
+                              'nuclear_kwargs': postprocess_kwargs_nuclear,
+                              'compartment': compartment}
```

Comment:
so `compartment` could be `nuclear`, `whole_cell` or `both`. If it is `both` then both kwargs are needed, and if it is one of the others, then only one kwargs is needed?

The post-processing will catch any bad values, correct?

Reply:
The kwargs aren't needed, since they default to be empty and all of the parameters in the actual function have default values which are only overriden in the case that they are supplied here. But yes, to override a value in nuclear predictions the nuclear_kwargs dict has to be provided.